src/main/java/hu/oe/nik/szfmv/automatedcar/Main.java

```diff
@@ -43,13 +43,14 @@ private void loop() {
         while (true) {
             try {
                 car.drive();
+
                 window.getCourseDisplay().drawWorld(world);
-//                window.getCourseDisplay().refreshFrame();
+                //window.getCourseDisplay().refreshFrame();
```

Comment:
Ha ez nem kell, akkor miért nincs kiszedve?

Reply:
Ezt még ti tettétek bele a kiinduló verzióba. A CourseDisplay.refreshFrame metódus nem is létezik. Gondolom egy javasolt ötlet akart lenni a vizualizációs team-nek. Valamiért a "//"-t elmozgattuk a sor elejéről ezért jelenik meg itt, érdemben nem változott a sor.

Reply:
Mikor én betettem, biztosan volt funkciója, mert fordult a kód. Lehet nem vettem észre a másik csapatnál, még kéne nézni a blame-et. 

Mindenesetre, ha valamit átírtok a kiadott kódban akkor a szemetet azért ki kell dobni. Amúgy bármi átírható, de nem kell ott hagyni csak azért mert mi írtuk.

Reply:
Ki volt már akkor is kommentezve, a legelején, azaz fordult.
Azért hagytam benne mert a visualisation csapathoz tartozik, nekik meg lehet hogy vmi hasznos ötletnek van itt hogy hogy csinálják.